lib/cc/engine/scalastyle.rb

```diff
@@ -57,22 +57,28 @@ def analysis_files
       end
 
       def scalastyle_jar
-        '/home/app/scalastyle_2.11-0.6.0-batch.jar'
+        '/home/app/scalastyle_2.11-0.8.0-batch.jar'
       end
 
       def scalastyle_config
-        @engine_config['config'] || '/home/app/scalastyle_config.xml'
+        @engine_config['config'] || '/code/scalastyle_config.xml'
```

Comment:
I'm either misunderstanding, or this isn't right. `/code` will be the project being analyzed. It won't contain the `scalastyle_config.xml` file (except when you run the engine on its own source).